src/renderer/js/main.js

```diff
@@ -48,6 +50,27 @@ ipcRenderer.on("window-is-focused", (event, data) => {
 document.addEventListener("click", event => {
   var target = event.target;
   while (target && target !== document) {
+    if (target.matches("a") || target.matches("button")) {
+      // TODO: Look into using accessiblity labels (this would also make the app more accessible)
+      let hrefParts = window.location.href.split("#");
+      let element = target.title || target.text.trim();
+      if (element) {
+        amplitude
+          .getInstance()
+          .logEvent("CLICK", {
+            target: element,
+            location:
+              hrefParts.length > 1 ? hrefParts[hrefParts.length - 1] : "/",
+          });
+      } else {
+        amplitude
+          .getInstance()
+          .logEvent("UNMARKED_CLICK", {
```

Comment:
Do you have an example of when this would fire? Just curious

Reply:
For instance, if there is an icon with no title attribute. I put this in there so, if we see it, we can go and add a title.

Reply:
👍